sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,16 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self.args[0].is_real:
+            return True
```

Comment:
But wouldn't that mean that we will be trying to solve inequality in complex domain? Which is neither supported in SymPy nor will make much sense. The inequality will be `t**2 + 1 > 0` where `t = exp(z)`.